### PR TITLE
refactor: office xml addin string and readme

### PR DESCRIPTION
**Pull request by ZYUN-MSFT**
This PR did the following:
1. modify create step by step office xml addin entry icon
2. modify create step by step office xml addin string
3. modify office xml addin readmes

### CODE REVIEW

File: packages/fx-core/resource/package.nls.json [hunk 1]
```diff
@@ -375,35 +375,35 @@
...
   "core.createProjectQuestion.officeXMLAddin.powerpoint.title": "PowerPoint Add-in",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.detail": "Build engaging solutions for presentations across platform",
-  "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.title": "Create a PowerPoint add-in with Single Sign On capabilities",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.title": "Add-in with Single Sign On",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.sso.detail": "PowerPoint add-in with Single Sign On capabilities",
-  "core.createProjectQuestion.officeXMLAddin.powerpoint.react.title": "Add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.react.title": "Add-in with React Framework",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.react.detail": "Create a PowerPoint add-in with React framework",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.create.title": "Create a PowerPoint Add-in",
   "core.createProjectQuestion.officeXMLAddin.taskpane.title": "Add-in with Basic Taskpane",
```
Comment (hermanwenhe) [inline packages/fx-core/resource/package.nls.json:404]: Taskpane -> Task Pane

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -375,35 +375,35 @@
...
   "core.createProjectQuestion.officeXMLAddin.taskpane.detail": "Customize the Ribbon with a button and embed content in the Taskpane",
```
Comment (hermanwenhe) [inline packages/fx-core/resource/package.nls.json:405]: Customize the Ribbon with a button and embed content in the Taskpane -> Customize the Ribbon with a button and create a dashboard in the Task Pane.

File: templates/js/office-xml-addin-powerpoint-sso/README.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ PowerPoint add-ins are integrations built by third parties into PowerPoint by us
 
 ## Instructions
 
+Before run and start the debug, make sure that:
+1. Close all Opened Office Application windows.
```
Comment (hermanwenhe) [inline templates/js/office-xml-addin-powerpoint-sso/README.md:13]: Opened -> opened

File: templates/js/office-xml-addin-powerpoint-sso/README.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ PowerPoint add-ins are integrations built by third parties into PowerPoint by us
...
+2. Click the *`Check and Install Dependencies`* in Teams Toolkit extension side bar.
```
Comment (hermanwenhe) [inline templates/js/office-xml-addin-powerpoint-sso/README.md:14]: side bar -> sidebar

File: packages/fx-core/resource/package.nls.json [hunk 5]
```diff
@@ -375,35 +375,35 @@
   "core.createProjectQuestion.officeXMLAddin.bar.detail": "Creating Project.",
   "core.createProjectQuestion.officeXMLAddin.mainEntry.title": "Office Add-in",
   "core.createProjectQuestion.officeXMLAddin.mainEntry.detail": "Create integration with Outlook, Word, Excel, or PowerPoint",
-  "core.createProjectQuestion.officeXMLAddin.create.title": "Select to create an Outlook, Word, Excel, or PowerPoint Add-in",
+  "core.createProjectQuestion.officeXMLAddin.create.title": "Select to Create an Outlook, Word, Excel, or PowerPoint Add-in",
   "core.createProjectQuestion.officeXMLAddin.word.title": "Word Add-in",
   "core.createProjectQuestion.officeXMLAddin.word.detail": "Create an add-in that can run in Word across multiple platforms",
   "core.createProjectQuestion.officeXMLAddin.word.sso.title": "Add-in with Single Sign On",
   "core.createProjectQuestion.officeXMLAddin.word.sso.detail": "Create a Word add-in with Single Sign On capabilities",
```
Comment (hermanwenhe) [inline packages/fx-core/resource/package.nls.json:382]: add-in -> Add-in

File: packages/fx-core/resource/package.nls.json [hunk 6]
```diff
@@ -375,35 +375,35 @@
...
-  "core.createProjectQuestion.officeXMLAddin.powerpoint.react.title": "Add-in with React framework",
+  "core.createProjectQuestion.officeXMLAddin.powerpoint.react.title": "Add-in with React Framework",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.react.detail": "Create a PowerPoint add-in with React framework",
   "core.createProjectQuestion.officeXMLAddin.powerpoint.create.title": "Create a PowerPoint Add-in",
-  "core.createProjectQuestion.officeXMLAddin.taskpane.title": "Add-in with Basic Taskpane",
-  "core.createProjectQuestion.officeXMLAddin.taskpane.detail": "Customize the Ribbon with a button and embed content in the Taskpane",
-  "core.createProjectQuestion.officeXMLAddin.manifestOnly.title": "Add-in project containing the manifest only",
+  "core.createProjectQuestion.officeXMLAddin.taskpane.title": "Add-in with Basic Task Pane",
+  "core.createProjectQuestion.officeXMLAddin.taskpane.detail": "Customize the Ribbon with a button and create a dashboard in the Task Pane",
+  "core.createProjectQuestion.officeXMLAddin.manifestOnly.title": "Add-in Project Containing the Manifest Only",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:406]: Proposed:
Add-in Project With only Manifest File
Logic: To be consistent with the description and also to follow the usual adjective-noun order in English and sounds more natural.

File: packages/fx-core/resource/package.nls.json [hunk 7]
```diff
@@ -375,35 +375,35 @@
...
   "core.createProjectQuestion.officeXMLAddin.manifestOnly.detail": "Create a simple add-in project with a manifest file only",
```
Comment (supkasar) [inline packages/fx-core/resource/package.nls.json:407]: Proposed: Create an add-in project that includes only the manifest file

**Review by supkasar — APPROVED** [review]

**Pull request merged**